### PR TITLE
Block tracking & Auto Pagination transform

### PR DESCRIPTION
**Pull request by ardatan**
Related: https://github.com/graphprotocol/graph-client/issues/18 https://github.com/graphprotocol/graph-client/issues/38

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@graphql-mesh/cli": "0.66.2",
+    "@graphql-mesh/cli": "0.67.0-alpha-e0a33e998.0",
```
Comment (dotansimha) [inline packages/cli/package.json]: Let's try to get this to stable version before merging here

File: examples/block-tracking/.graphclientrc.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+sources:
+  - name: uniswap
+    handler:
+      graphql:
+        endpoint: https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2
+    transforms:
+      # Enable Automatic Block Tracking
+      - blockTracking:
```
Comment (dotansimha) [inline examples/block-tracking/.graphclientrc.yml:8]: I love it

File: packages/cli/package.json (hunk 1 above)
Comment (ardatan) [reply packages/cli/package.json]: Done!

File: README.md [hunk 3]
```diff
@@ -281,6 +282,24 @@ sources:
...
+The Graph Client can track block numbers and do the following queries by following [this pattern](https://thegraph.com/docs/en/developer/distributed-systems/#polling-for-updated-data) with `blockTracking` transform;
+
+```yaml
+sources:
+  - name: uniswapv2
+    handler:
+      graphql:
+        endpoint: https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2
+    transforms:
+      - blockTracking:
```
Comment (dotansimha) [inline README.md:296]: @ardatan since this is applied on the source level, maybe we can add a list of operationNames to ignore? 🤔

Comment (ardatan) [reply README.md:296]: Yes! It is implemented now!

**Pull request merged**